lib/btn/btn.scss

```diff
@@ -0,0 +1,36 @@
+@import "palette.scss";
+@import "mixins/btn.scss";
+
+//button variants
+.btn {
+  @include btn();
+}
+
+.btn--light {
+  @include btn($color-400-grey);
+}
+
+.btn--active {
+  @include btn($btn-active-bg-color);
+}
+
+.btn--danger {
+  @include btn($danger-color);
+}
+
+.btn--mortal-danger {
+  @include btn($mortal-danger-color);
+}
+
+//button sizes
+.btn--small {
+  @include btn-size("small");
+}
+.btn--large {
+  @include btn-size("large");
+}
+
+.btn--very-large {
```

Comment:
unfortunately we a coupled to the composer application as this should be a drop in replacement.  If composer used sass then we could just re use the mixins but it isn't so we have to go with what we have got as I don't want to go through the whole application and re-name classes.